net.hachune.mikutter.appdata.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It should be maintained in upstream repo so other distributions can use it too.

Reply:
I'll contact upstream to include appdata file.